data/map.txt

```diff
@@ -27028,21 +27028,35 @@ planet "Elek Kartanu"
 
 planet "Ember Graveyard"
 	attributes "requires: quantum keystone"
-	description ""
+	wormhole
+		linked
+		link Ossipago Polerius
+		link Polerius Ossipago
```

Comment:
> as that definitely isn't backwards compatible

No?

> there's no way to inject the required data into already-processed or pending GameEvents

There's no need to do that. The Eye and the Pug Wormhole work, because the wormhole needs to be added as an object to the systems in question so that you can use them. Without the event, then the planets are just unused.